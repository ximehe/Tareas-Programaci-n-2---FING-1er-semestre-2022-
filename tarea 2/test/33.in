# implementar reiniciarIterador
reiniciarIterador
estaDefinidaActual
agregarAIterador 5
reiniciarIterador
estaDefinidaActual
agregarAIterador 5
agregarAIterador 5
agregarAIterador 5
reiniciarIterador
estaDefinidaActual
Fin