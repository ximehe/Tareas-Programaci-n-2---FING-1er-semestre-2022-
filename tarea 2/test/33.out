1>#  implementar reiniciarIterador.
2>Reiniciando it.
3>Actual de it NO está definida.
4>Agregando 5 a it.
5>Reiniciando it.
6>Actual de it está definida.
7>Agregando 5 a it.
8>Agregando 5 a it.
9>Agregando 5 a it.
10>Reiniciando it.
11>Actual de it está definida.
12>Fin.
